src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -67,8 +67,6 @@ public function render(Message $message): void
             $message->htmlTemplate(null);
         }
 
-        $message->context([]);
```

Comment:
That's useful, see https://github.com/symfony/symfony/pull/47075#discussion_r930788561 for the explanation of why we need to empty the context.

Reply:
We should probably have a test covering that use case, to avoid regressions on that use case

Reply:
Interesting, thanks for clarifying, I hadn't seen that.
I wonder whether by doing this we end up removing the context from the debug profiler as well though. I wonder if there is a way to keep this information in the logs/profiler.

Reply:
Would it perhaps be preferable to clear the context at the time of serializing for the messenger dispatch, perhaps by cloning the message and clearing?

Reply:
Omitting the context when serializing a TemplatedEmail that is already rendered might indeed be an option.

Reply:
OK cool, as I'm working in test environment for behat functional, I don't have a test which triggers the PhpSerializer to serialize the envelope. I'll try and work out how to make a failing test when the context is not cleared first and then where (that would make sense) to hook in the logic to check for a TemplatedEmail before the serializer is called.

Reply:
I've updated the PR request to clear context before sending to the bus instead as an alternative option.

Reply:
Your change would break valid use cases. There are 2 supported use cases: rendering emails before sending them to the bus and rendering them when consumed from the queue. So, we can only clear the context when the email is rendered, which is why the current code is here.

Reply:
Is there another alternative? This code change would break tests so would the advice to anyone upgrade from 6.1 to 6.2 be to adjust their tests and perhaps am I wrong in thinking that by clearing the context the profiler would not longer be able to output the context set in the email too?
Would the change in RC1 as it stands not constitute a breaking change?